spring-integration-debezium/src/main/java/org/springframework/integration/debezium/dsl/Debezium.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.debezium.dsl;
+
+import java.time.Clock;
+import java.util.Properties;
+
+import javax.annotation.Nullable;
+
+import io.debezium.engine.ChangeEvent;
+import io.debezium.engine.DebeziumEngine;
+import io.debezium.engine.DebeziumEngine.Builder;
+import io.debezium.engine.DebeziumEngine.CompletionCallback;
+import io.debezium.engine.DebeziumEngine.ConnectorCallback;
+import io.debezium.engine.format.KeyValueHeaderChangeEventFormat;
+import io.debezium.engine.format.SerializationFormat;
+import io.debezium.engine.spi.OffsetCommitPolicy;
+
+import org.springframework.util.Assert;
+
+/**
+ * Factory class for Debezium components DSL.
+ *
+ * @author Christian Tzolov
+ *
+ * @since 6.2
+ */
+public final class Debezium {
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided {@link DebeziumEngine.Builder}.
+	 * @param debeziumEngineBuilder the {@link DebeziumEngine.Builder} to use.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(
+			Builder<ChangeEvent<byte[], byte[]>> debeziumEngineBuilder) {
```

Comment:
I would prefer a `DebeziumEngine.Builder` style: better for review and readability.